src/person/person.interface.ts

```diff
@@ -3,13 +3,13 @@ import { Rank, Responsibility } from '../utils';
 import { IOrganizationGroup } from '../group/organizationGroup/organizationGroup.interface';
 import { ObjectId } from 'bson';
 
-export interface IUser extends Document {
-// User's Basic information
+export interface IPerson extends Document {
+// Person's Basic information
   _id: ObjectId;
   identityCard: string;
   personalNumber: string;
-  primaryUser: string;
-  secondaryUsers: string[];
+  primaryPerson: string;
+  secondaryPersons: string[];
```

Comment:
Should be "domainUsers" or "secondaryDomainUsers"

Reply:
Done

Reply:
Done